### DISCUSSION TITLE
CommandStream used too much space

### DISCUSSION THREAD

**Discussion by simonsland** [google/filament#5427] [Q&A]
<img width="756" alt="截屏2022-04-13 14 43 22" src="https://user-images.githubusercontent.com/17590947/163116250-7b988ee2-c7c8-4c87-ac7b-f4a8226e431f.png">
when I render a glb model, I got the above log. Is it because the mesh of the model too complex? And should I modify the value of FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB?

**Comment by bejado** [comment]
Yes, building Filament with a larger value of `FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB` should help alleviate the issue. This problem is generally when when rendering models with lots of individual renderables.

**Reply by HappySapeta** [reply]
This issue still occurs with the latest version of Filament, built for Windows with vs2022 with the OpenGL backend.
I have tried several higher values for FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB - 2, 4 and 32 but the warnings do not go away. 

This occurs when Filament is built in Debug mode only. (Or maybe its just that the logs appear in Debug mode)

**Reply by BStringhamVRSK** [reply]
See #7968 for a possible solution, if the problem is that you are sending too many frames before they can be processed.
